src/kv/RocksDBStore.cc

```diff
@@ -491,6 +494,11 @@ int RocksDBStore::submit_transaction(KeyValueDB::Transaction t)
 
 int RocksDBStore::submit_transaction_sync(KeyValueDB::Transaction t)
 {
+  RocksDBTransactionImpl * _t =
```

Comment:
sometimes we might need a empty sync transaction to make sure previous transactions are flushed to the db. After a batch of non sync transaction, we will do a sync transaction to make sure previous one made to db. It used to be the case in kv_sync_thread before. Could we make sure we covered all those scenarios.